PythonTool/MakeImagesTerminal.py

```diff
@@ -313,4 +313,4 @@
 
 toc = timeit.default_timer()  # End timer.
 ElapsedTime = toc - tic
-print "Elapsed time was "+str(ElapsedTime)+" seconds."
+print "Elapsed time was " +str(ElapsedTime)+ " seconds."
```

Comment:
additional pep8: should have spaces on both sides of the +

Reply:
Also, with updating this to work with Python3, should use `print()` instead of `print`